travel/travel.py

```diff
@@ -25,6 +25,17 @@
 from .res_config import get_basic_passenger_limit
 
 
+def _get_travel_states(self, cr, uid, ids=None, context=None):
```

Comment:
@pedrobaeza This is not a bug in the ORM. The ORM works as designed, with its limitations.